CMakeLists.txt

```diff
@@ -107,7 +110,7 @@ endif()
 add_subdirectory(sentencepiece sentencepiece EXCLUDE_FROM_ALL)
 
 add_library(tokenizers_c INTERFACE ${TOKENIZERS_RUST_LIB})
-target_link_libraries(tokenizers_c INTERFACE ${TOKENIZERS_RUST_LIB})
+target_link_libraries(tokenizers_c INTERFACE ${TOKENIZERS_RUST_LIB} ${TOKENIZERS_C_LINK_LIBS})
```

Comment:
actually we can rename `TOKENIZERS_CPP_LINK_LIBS ` to TOKENIZERS_C_LINK_LIBS as they are all intended to fix the rust lib deps

Reply:
Yeah, `TOKENIZERS_CPP_LINK_LIBS`  can be replaced with `TOKENIZERS_C_LINK_LIBS` in this case. I think the `TOKENIZERS_CPP_LINK_LIBS` maybe also kept empty for extension of some other potential cpp extra libraries.

Reply:
What I meant is that we should move some of the `TOKENIZERS_CPP_LINK_LIBS ` deps to TOKENIZERS_C_LINK_LIBS as they are related to C lib really


https://github.com/mlc-ai/tokenizers-cpp/blob/main/CMakeLists.txt#L20

and here